client/src/modules/stock/lots/modals/search.modal.html

```diff
@@ -11,58 +11,83 @@
     </ol>
   </div>
 
-  <div class="modal-body">
+  <div class="modal-body" style="overflow: auto; max-height: 600px; padding: 0px">
+    <uib-tabset>
+      <uib-tab index="0" heading="{{ 'FORM.LABELS.SEARCH_QUERRIES' | translate}}" data-custom-filter-tab>
+        <div class="tab-body">
+          <!-- depot -->          
+          <bh-depot-select
+            depot-id="$ctrl.searchQueries.depot_uuid"
```

Comment:
Could you rename this parameter `depot-uuid`?  It will save confusion in the future.  Note that the `name` property of this component will also need to be updated.